### DISCUSSION TITLE
Add option to specify default basePath

### DISCUSSION THREAD

**Discussion by kiliman** [kiliman/remix-flat-routes#3] [Ideas]
I have another package `remix-mount-routes` to be able to mounts using a different basePath from `/`, such as `/myapp`.

Perhaps include an option to `flatRoutes` to enable this as well.

```ts
flatRoutes('routes', defineRoutes, { basePath: '/myapp' })
```

**Comment by kiliman** [comment]
Added this in v0.3.0

**Comment by rjstanford** [comment]
@kiliman can I ask a stupid user question?

I installed the package and have this in my config file:
```
  routes: (defineRoutes) => {
    return flatRoutes("routes", defineRoutes, { basePath: "/admin-center" });
  },
```
no errors but no real change either, continues to work the same locally as it did before, and when I access it through my CDN at "site.com/admin-center/" (maps to the running image at /), while the site loads its still trying to reference files like `/build/root-7JQY2EF2.js` (in other words, basepath doesn't seem to be being used).

Any chance I'm missing something obvious?

**Reply by kiliman** [reply]
You also need to update `publicPath` in _remix.config.js_

https://remix.run/docs/en/main/file-conventions/remix-config#publicpath
